test/unit/socketAuthTest.php

```diff
@@ -15,12 +15,19 @@ public function testObjectConstruct()
 
 		public function testSocketAuthKey()
 		{
-			$socket_auth = $this->pusher->socket_auth('testing_pusher-php', 'testing_socket_auth');
+			$socket_auth = $this->pusher->socket_auth('testing_pusher-php', '1.1');
```

Comment:
For clarity (when considering tests as specs) it may be worth having a few more examples of valid socket_id values.